apis/placement/v1/clusterresourceplacement_types.go

```diff
@@ -513,7 +513,7 @@ type RollingUpdateConfig struct {
 	// The desired number equals to the number of clusters scheduler selected when the placement type is `PickAll`.
```

Comment:
// The desired number equals to the `NumberOfClusters` field when the placement type is `PickN`.

What is the desired number?